lnbits/core/migrations.py

```diff
@@ -684,3 +684,22 @@ async def m029_create_audit_table(db: Connection):
         );
         """
     )
+
+
+async def m030_add_currency_to_wallet(db: Connection):
+    """
+    Setting currency to default is heavy, so easier just to add a trigger.
+    """
+    await db.execute(
+        """
+        CREATE TRIGGER IF NOT EXISTS set_default_currency
```

Comment:
i am not sure about a db trigger for default values, we do not handle defaults like that anywhere, why not setting the default on for example `create_wallet` function?

Reply:
yeah thats a better idea